subprojects/android-test/test-report/src/main/kotlin/com/avito/android/test/report/ReportImplementation.kt

```diff
@@ -254,9 +254,8 @@ class ReportImplementation(
     override fun addHtml(label: String, content: String, wrapHtml: Boolean) {
```

Comment:
Do we still need `addHtml` with `wrapHtml = false`?

Reply:
I don't know It hard for me to decide in that PR
Let's look further and then remove this parameter